xml/System.Buffers/SequenceReader`1.xml

```diff
@@ -263,8 +263,8 @@
         <ReturnType>System.ReadOnlySpan&lt;T&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets a <see cref="T:System.Span`1" /> that contains the current segment in the <see cref="P:System.Buffers.SequenceReader`1.Sequence" />.</summary>
+        <value>A span that contains the current segment in the sequence.</value>
```

Comment:
It's a bit odd to read that a span contains anything since it just represents the view.

How it was written in source makes more sense to me:
https://github.com/dotnet/corefx/blob/fec76b0fc06b4fce9e4ac28b9a2bf1101209d984/src/System.Memory/src/System/Buffers/SequenceReader.cs#L59
```text
The current segment in the <see cref="Sequence"/> as a span.
```

Maybe:
```text
A view of the current segment in the sequence, as a span.
```

cc @JeremyKuhne